src/.nuget/NuGet.Config

```diff
@@ -6,6 +6,8 @@
     <clear />
     <add key="myget.org buildtools" value="https://www.myget.org/F/dotnet-buildtools/" />
     <add key="myget.org dotnet-corefx" value="https://www.myget.org/F/dotnet-corefx/" />
+    <add key="myget.org nugetbuild" value="https://www.myget.org/F/nugetbuild/api/v3/index.json" />
+    <add jey="nuget.org v2" value="https://www.nuget.org/api/v2/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
I specify it on the command line. If you know why I should not, please let me know.